### PR TITLE
[chunky_png] Added RBS files for Chunky PNG

### PR DESCRIPTION
**Pull request by raosush**
# Added RBS file for [Chunky PNG](https://github.com/wvanbergen/chunky_png) gem - v1.4.0

- Created gem directory & corresponding version directory
- Added RBS file in the correctly versioned directory
- Added inline documentation from the source code for each method in the RBS file
- Steepfile has been added in the `_test` directory
- 5 tests added in the `_test` directory
- `test` script added in `_scripts` directory
- `git submodule` pointing to the source code of the gem has been added

* Followed the syntax given in [RBS repo](https://github.com/ruby/rbs/blob/master/docs/syntax.md)
* Followed the guidelines given in [RBS Wiki](https://github.com/ruby/rbs/wiki/Writing-Signatures-Best-Practices)

Signed-off-by: Sushanth Sathesh Rao <rao.sush004@gmail.com>

### CODE REVIEW

File: gems/chunky_png/1.4.0/chunky_png.rbs [hunk 1]
```diff
@@ -0,0 +1,1362 @@
...
+    def edges: (?bool close) -> Enumerator[untyped, void]
+
+    # Returns the number of points in this vector.
+    # @return [Integer] The length of the points array.
+    def length: () -> Integer
+
+    # Iterates over all the points in this vector
+    # @yield [ChunkyPNG::Point] The points in the correct order.
+    # @return [void]
+    def each: () { () -> untyped } -> void
```
Comment (soutaro) [inline gems/chunky_png/1.4.0/chunky_png.rbs:241]:
```suggestion
    def each: () { (ChunkyPNG::Point point) -> void } -> void
```

File: gems/chunky_png/1.4.0/chunky_png.rbs [hunk 2]
```diff
@@ -0,0 +1,1362 @@
...
+
+    # Comparison between two vectors for quality.
+    # @param [ChunkyPNG::Vector] other The vector to compare with.
+    # @return [true, false] true if the list of points are identical
+    def eql?: (instance other) -> bool
```
Comment (soutaro) [inline gems/chunky_png/1.4.0/chunky_png.rbs:246]: I usually use `untyped` for the argument of equality operator, but the specific one is correct in this case. The implementation doesn't check the class of `other` value.

File: gems/chunky_png/1.4.0/chunky_png.rbs (hunk 1 above)
Comment (raosush) [reply gems/chunky_png/1.4.0/chunky_png.rbs]: Sir, I updated the return type of the block & changed the type of the parameter, hence resolving the conversation.

File: gems/chunky_png/1.4.0/chunky_png.rbs (hunk 2 above)
Comment (raosush) [reply gems/chunky_png/1.4.0/chunky_png.rbs]: Sir, I update the type of the parameter from `instance` to `untyped`, hence resolving the conversation.

**Review by soutaro — APPROVED** [review]
👏

**Pull request merged**